### DISCUSSION TITLE
How are files loaded into the game? (SSD Degradation concern)

### DISCUSSION THREAD

**Discussion by Kylie-Digital** [xivdev/Penumbra#183] [Q&A] [answered]
Hi My main question is this, How exactly are mod files loaded into the game?

My concern is a heavily modded setup could be writing over the base game files, and for SSD users could be a heavy cost long-term. 
_I'm hoping Penumbra re-directs files to be loaded from it's mod folder though and doesn't actively Write anything normally._

**Comment by avafloww** [comment]
![image](https://user-images.githubusercontent.com/654783/162563168-ad63d060-9f61-479f-a56b-57152e588d90.png)

**Reply by Kylie-Digital** [reply]
Okay, So Files are just loaded from penumbra's directory as needed on demand?

**Comment by reiichi001** [comment, marked as answer]
Yes. Mods are loaded from whatever you've defined as the "Root Directory" in the plugin settings. It can be stored anywhere, but you probably want to avoid putting it onto slow or external media when possible. 

It is 100% portable and be moved around if wanted.

Unlike textools, mods are fully unpacked so that you can do whatever you want with the raw files.

**Reply by Kylie-Digital** [reply]
Awesome! really happy to hear that, Thank you.
